### ISSUE TITLE
[BUG] node_exporter service not restarted on archive fresh install

### ISSUE THREAD

**Issue by n-rodriguez** [saltstack-formulas/prometheus-formula#42]
### Pillar / config used

---

### Describe the bug

```sh
[DEBUG   ] Could not LazyLoad service.mod_run_check: 'service.mod_run_check' is not available.
[INFO    ] Executing command 'systemctl list-units | grep node_exporter >/dev/null 2>&1' in directory '/home/kitchen'
[DEBUG   ] retcode: 1
[DEBUG   ] Last command return code: 1
[INFO    ] ['onlyif condition is false']
[INFO    ] Completed state [node_exporter] at time 14:17:22.658642 (duration_in_ms=983.05)
[INFO    ] Running state [node_exporter] at time 14:17:22.660463
[INFO    ] Executing state service.running for [node_exporter]
[DEBUG   ] Could not LazyLoad service.mod_run_check: 'service.mod_run_check' is not available.
[INFO    ] Executing command 'systemctl list-units | grep node_exporter >/dev/null 2>&1' in directory '/home/kitchen'
[DEBUG   ] retcode: 1
[DEBUG   ] Last command return code: 1
[INFO    ] ['onlyif condition is false']
```

```sh
kitchen@d22a0b418b50:~$ sudo -s
root@d22a0b418b50:/home/kitchen# systemctl list-units | grep node_exporter
root@d22a0b418b50:/home/kitchen# systemctl list-unit-files | grep node_exporter
node_exporter.service                  disabled        enabled      
root@d22a0b418b50:/home/kitchen# 
```

**Comment by javierbertoli** [comment]
@n-rodriguez I think the issue is related to the 'assumed' names the formula uses: debian's packages/services are named `prometheus-component{-exporter}` and the formula assumes `component{_exporter}`, ie:

Debian's packages (and services): prometheus-alertmanager, prometheus-node-exporter
Formula packages (and services): alertmanager, node_exporter

I think these are fixed in #44. If you want to give them a try

**Comment by n-rodriguez** [comment]
> I think these are fixed in #44. If you want to give them a try

It won't work : for instance, [here](https://travis-ci.com/github/n-rodriguez/prometheus-formula/jobs/377065781#L2803) it fails because of the wrong service name due to https://github.com/saltstack-formulas/prometheus-formula/blob/master/prometheus/defaults.yaml#L136

**Comment by n-rodriguez** [comment]
with all this info you should now understand what I meant here : https://github.com/saltstack-formulas/prometheus-formula/issues/35
